test/propagatortest.jl

```diff
@@ -127,3 +127,8 @@ end
 
 ind = findin(tlist, rand(tlist))[1]
 @test_approx_eq_eps coeffs(vec(rhos[ind])) coeffs(vec(ps[ind])) 1e-1
+
+# tests for  evolution opeator..
```

Comment:
I think it is better to apply the QuPropagator to an initial state and compare that to a state propagated with some other method (maybe Expokit).